.tekton/operator-1-18-operator-push.yaml

```diff
@@ -37,7 +37,7 @@ spec:
     - linux/x86_64
   - name: prefetch-input
     value: |
-      {"type": "rpm", "path": ".konflux/prefetch"}
```

Comment:
@pramodbindal i think we need to handle this from hack :thinking: its only for Operator

cc @vdemeester

Reply:
It is only for bundle. So either we move all rpms to prefetch everywhere or we move back rpm at its old place here.